loops-and-arrays.js

```diff
@@ -3,36 +3,44 @@
 // Schrijf een script dat iedere naam in onderstaande array vervangt door een koosnaampje, door er -"je" achter te plakken.
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
-
 const names = ["Henk", "Piet", "Fred", "Joop"];
+console.log(names);
+
+for(let i = 0; i < names.length; i++) {
+    names[i] = names[i] + "je";
+}
+
+console.log(names);
 
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
-
-
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
-
 const numbers = [2, 4, 5, 29, 38];
+console.log(numbers);
+
+for(let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+        numbers[i] = numbers[i] * 2;
+    } else if (numbers[1] % 1 === 0) {
```

Comment:
Volgens mij moet je hier ipv number[1] , numbers [ i ] zetten. 
Daarbij denk ik, maar correct me if im wrong, dat je ipv de "else if" ook gewoon een else statement had kunnen maken met  daarin de "numbers[i] = numbers[i] * 3"  in dit specifieke geval tenminste

Reply:
Hi Melroy, goed gezien! Ik maak vaak de typo 1 ipv i. Wel bijzonder dat de antwoorden alsnog gewoon kloppen. En die verkorting werkt inderdaad ook dus goede feedback dankjewel!